yoti/tests/yoti.test

```diff
@@ -273,21 +274,9 @@ class YotiTest extends DrupalWebTestCase {
    * Test Unlink Form.
    */
   public function testUnlinkForm() {
-    // Log in as unlinked user.
-    $this->drupalLogin($this->unlinkedUser);
-
-    // Check selfie exists and cannot be viewed by another user.
-    $this->assertTrue(is_file($this->selfieFilePath), 'Check selfie exists');
-    $this->drupalGet('yoti/bin-file', array('query' => array('field' => 'selfie')));
-    $this->assertNoRaw('test_selfie_contents');
-
```

Comment:
We're not removing these checks, we've just moved them to `testProfile`, right?

Reply:
Yes, but the response is now returning 404 as a result of the token not being present in the URL.

Do you think I should add an extra test to ensure that any user with "view user profiles" permission can also see the selfies? (I did this for the Drupal 8 module)

Reply:
Added coverage in d914717

Reply:
Nice 👌